build/common-config.sh

```diff
@@ -37,11 +37,18 @@ var_set() {
 
 	env_val[WASM_SRC]="${WASM_SRC:-${PDO_SOURCE_ROOT}/interpreters/wasm-micro-runtime}"
 	env_desc[WASM_SRC]="
-		WASM_SRC points to the installation of the micro-wasm
+		WASM_SRC points to the installation of the wasm micro runtime
 		source in order to build the wasm interpreter
 	"
 	env_key_sort[$i]="WASM_SRC"; i=$i+1; export WASM_SRC=${env_val[WASM_SRC]};
 
+        env_val[WASM_MODE]="${WASM_MODE:-INTERP}"
+	env_desc[WASM_MODE]="
+		WASM_MODE indicates whether to build the wasm runtime in intepreted
+                or aot-compiled mode
```

Comment:
might be worth mentioning the exact values you expect here?

Reply:
Yes, good point.